vaadin-upload-flow-parent/vaadin-upload-flow-integration-tests/src/test/java/com/vaadin/flow/component/upload/tests/UploadI18nIT.java

```diff
@@ -112,6 +113,28 @@ public void testPartialI18nShouldSetFullConfigurationWithoutNullValuesInWebCompo
         assertTranslationMapHasNoMissingTranslations(translationMap);
     }
 
+    @Test
+    public void testDetachReattachI18nIsPreserved() {
+        open();
+
+        NativeButtonElement btnSetI18n = $(NativeButtonElement.class)
+                .id("btn-set-i18n");
```

Comment:
Since the test only needs the buttons to perform the click action, this could be done with:
```suggestion
        var btnSetI18n = findElement(By.id("btn-set-i18n"));
```

Doing so, importing the `flow-html-components-testbench` module wouldn't be needed.

Reply:
Thanks, done.